### ISSUE TITLE
E117: Unknown function: UpdateDiags

### ISSUE THREAD

**Issue by Shane-XB-Qian** [yegappan/lsp#27]
```log
lsp/autoload/handlers.vim[22]..function diag#DiagNotification:
line   24:
E117: Unknown function: UpdateDiags
```

recently vim and this changed much, not some err not very sure what happened... :-)

**Comment by yegappan** [comment]
This is caused by a bug in the Vim9 import support (recent changes made to the
Vim9 import feature). I have pushed 85417435fafc78c09d5a445690afe87210b3b7d7 to workaround
this problem. Let me know if this addresses this issue.

**Comment by yegappan** [comment]
This Vim bug has now been addressed by patch 8.2.4063 (b8822442d716df0230c79531132e530e95cc17e3).

**Comment by Shane-XB-Qian** [comment]
> I have pushed [8541743](https://github.com/yegappan/lsp/commit/85417435fafc78c09d5a445690afe87210b3b7d7) to workaround
> this problem. Let me know if this addresses this issue.

yes, that patch can workaround this issue. 
// thx. that vim9script 'import' refactor caused lots change...
